src/main/java/org/ibp/api/rest/germplasmlist/GermplasmListResourceGroup.java

```diff
@@ -65,5 +64,63 @@
 		return new ResponseEntity<>(germplasmListTypes, HttpStatus.OK);
 	}
 
+	@ApiOperation(value = "Create germplasm list folder", notes = "Create sample list folder.")
+	//TODO: we need to add GERMPLASM_LIST permission
+	@PreAuthorize("hasAnyAuthority('ADMIN','GERMPLASM','MANAGE_GERMPLASM','GERMPLASM_LIST')")
```

Comment:
Given Mariano comment today I would rewrite the TODO, because we dont know if we will create a more general permission. Should we add search permission too?